javascript/filter.js

```diff
@@ -0,0 +1,24 @@
+function filterSelection(section) {
+  const portfolioElements = document.getElementsByClassName("portfolio__box");
+
+  for (let i = 0; i < portfolioElements.length; i++) {
+    const currentElement = portfolioElements[i];
+
+    if (currentElement.classList.contains(section) || section === "") {
+      currentElement.classList.add("portfolio__show");
+    } else {
+      currentElement.classList.remove("portfolio__show");
+    }
+  }
+  portfolioSwiper.update();
+}
+
+const btnContainer = document.getElementsByClassName("portfolio__btns");
+const btns = btnContainer[0].getElementsByClassName("portfolio__btn");
+for (let i = 0; i < btns.length; i++) {
+  btns[i].addEventListener("click", function () {
+    let activeButton = document.getElementsByClassName("btn__active");
+    activeButton[0].classList.remove("btn__active");
+    this.classList.add("btn__active");
```

Comment:
Const here. Please read this [stackoverflow answer](https://stackoverflow.com/a/44604402/13840636) that explains constants with objects